faf-java-server-app/src/main/java/com/faforever/server/game/GameService.java

```diff
@@ -714,6 +714,10 @@ private void removePlayer(Game game, Player player) {
 
     clientService.disconnectPlayerFromGame(player.getId(), connectedPlayers.values());
 
+    if (game.getState() == GameState.OPEN && game.getHost().equals(player)) {
+      new ArrayList<>(connectedPlayers.values()).forEach(connectedPlayer -> removePlayer(game, connectedPlayer));
```

Comment:
values() is a a array and there is some Method like Arrays.asStream(). I think that that might be better as no ArrayList needs to be initialized. I know it does not really matter but I wanna be picky...

Reply:
It is called Arrays.stream(connectedPlayers.values())

Reply:
`values()` returns a collection but since you can't remove elements from the very collection your iterating over (ConcurrentModificationException), I need to create a copy.

I wasn't sure if I need to add a comment but apparently it's better if I do :-)